samples/project-anfield/import_cluster_config.py

```diff
@@ -293,6 +326,50 @@ def create_sources(source, environment, node):
                 imported_res_dict["Protection Sources"].append(name)
             return
 
+        elif environment == 'kCassandra':
+            name = source.protection_source.name
+            source_id = source.protection_source.id
+            ssh_username = configparser[name].get('username', None)
+            ssh_password = configparser[name].get('password', None)
+            db_username = configparser[name].get('db_username', None)
+            db_password = configparser[name].get('db_password', None)
+            if not (ssh_password and ssh_username):
+                ERROR_LIST.append(
+                    "Please provide ssh credentials for %s" % name)
+                return
+            if not (db_password and db_username):
+                ERROR_LIST.append(
+                    "Please provide database credentials for %s" % name)
+                return
+            _cassandra_params = node["registrationInfo"]["cassandraParams"]
+            cassandra_params = dict(
+                configDirectory=_cassandra_params["configDirectory"],
+                sshPrivateKeyCredential=None,
+                jmxCredentials=None,
+                dataCenterNames=_cassandra_params["dataCenters"],
+                dseConfigurationDirectory=_cassandra_params[
+                    "dseConfigDirectory"],
+                isDseAuthenticator=_cassandra_params["isDseAuthenticator"],
+                isDseTieredStorage=_cassandra_params["isDseTieredStorage"],
+                dseSolrInfo= None,
+                kerberosPrincipal= None)
+            cassandra_params["seedNode"] = source.registration_info.\
+                access_info.endpoint
+            cassandra_params["sshPasswordCredentials"] = dict(
+                    username=ssh_username, password=ssh_password)
+            cassandra_params["cassandraCredentials"] = dict(
+                username=db_username, password=db_password)
+            body = dict(
+                environment="kCassandra",
+                cassandraParams=cassandra_params)
+            api = 'data-protect/sources/registrations'
+            status_code, resp = rest_obj.post(
+                api, version='v2', data=json.dumps(body))
+            if status_code == 201:
```

Comment:
if status code is non 201 , then are we putting this part of ERROR_LIST ?

Reply:
Incase source registration is failed we are adding the error to ERROR_LIST